src/main/java/servlets/SSOLogin.java

```diff
@@ -0,0 +1,68 @@
+package servlets;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.apache.log4j.Logger;
+
+import com.onelogin.saml2.Auth;
+import com.onelogin.saml2.exception.Error;
+import com.onelogin.saml2.exception.SettingsException;
+import utils.ShepherdLogManager;
+
+/**
+ * Does SSO login. <br/>
+ * <br/>
+ * This file is part of the Security Shepherd Project.
+ * 
+ * The Security Shepherd project is free software: you can redistribute it
+ * and/or modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.<br/>
+ * 
+ * The Security Shepherd project is distributed in the hope that it will be
+ * useful, but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General
+ * Public License for more details.<br/>
+ * 
+ * You should have received a copy of the GNU General Public License along with
+ * the Security Shepherd project. If not, see <http://www.gnu.org/licenses/>.
+ * 
+ * @author Mark Denihan
+ *
+ */
+public class SSOLogin extends HttpServlet {
+
+	private static final long serialVersionUID = 1488140446224058032L;
+	private static org.apache.log4j.Logger log = Logger.getLogger(Login.class);
```

Comment:
Yup

Reply:
This has been fixed in a later PR

Reply:
Are the changes in this PR included in a later PR? Should this PR be closed? 

It takes a bit of time to review these PR's because of all the formatting churn.

Reply:
Yes, all my PR:s build on each other. Not sure if this is the right way to do things, please let me know otherwise. (I'm on the slack channel if you want to chat over a less formal channel). I'd say jump into PR #531 and you can see latest version.

Reply:
It would be more typical for each PR to be modular, but we don't have any formalized rules for PRs so it's not a problem. If all of the changes for your PR's are also in 531, please close the others and i'll start reviewing 531 instead